index.coffee

```diff
@@ -7,27 +7,28 @@ basicAuthParser = require 'basic-auth-parser'
 MiddlewareHandler = require 'middleware-handler'
 MiddlewareHandler.prototype = Promise.promisifyAll(MiddlewareHandler.prototype)
 
-# Simplify testing
-exports.connect = net.connect
```

Comment:
This is strictly speaking a breaking change, as previously this could be used to provide a custom connect function

Reply:
I wondered about this - updated to `change-type: major`.